src/cpu/x64/brgemm/jit_brdgmm_kernel.cpp

```diff
@@ -1007,8 +1007,8 @@ void jit_brdgmm_kernel_base_t<Wmm>::brdgmm_microkernel(int m_blocks,
 
             align(64);
             L(jmp_table_base);
-            for (int m_i = 0; m_i < m_blocks; ++m_i) {
-                putL(jmp_table_labels[m_i]);
+            for (const auto &label : jmp_table_labels) {
+                putL(label);
```

Comment:
Random spot: Could you keep the header of the commit message to 72 chars? Maybe consider the below:

```
cpu: x64: brgemm: use range-based loop
```

Reply:
Would you like us to amend the actual commit, or will the 'squash+merge' re-write from you merging this pull-request be sufficient?

Reply:
Good question. @mgouicem what would your team prefer for those cases?

Reply:
We generally prefer nice commit history in original PRs. While squash+merge is possible sometimes things slip by.

Reply:
OK! new title is 64 characters!

Reply:
Thanks.